### PR TITLE
added area (=0) attribute to Segment2D class

### LINKED ISSUE #20460: Triangle().area for zero-area triangle results in AttributeError
`Triangle(Point(0,0),Point(0,1),Point(0,0.5)).area`
results in 
`---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-11-4686e8494747> in <module>
----> 1 Triangle(Point(0,0),Point(0,1),Point(0,0.5)).area

AttributeError: 'Segment2D' object has no attribute 'area'`

I'm new to the project and understand that there may be some good reasons to cast a zero-area Triangle to a Segment2D; I would also strongly argue that Triangle() and its functions should work correctly for arbitrary user-inputted or randomly-generated points. Perhaps a simple fix is to implement Segment2D.area and any other functions for finite-area polygons?

### PR DESCRIPTION
**Pull request by KatsuragiCSL**
#### References to other Issues or PRs
Fixes #20460 

#### Brief description of what is fixed or changed
added area (=0) attribute to Segment2D class

NO ENTRY

### CODE REVIEW

**Comment by oscarbenjamin** [comment]
Looks like the tests have failed because of trailing whitespace:
```
AssertionError: File contains trailing whitespace: /home/travis/build/sympy/sympy/sympy/geometry/line.py, line 2369.
```

File: sympy/geometry/line.py [hunk 1]
```diff
@@ -2365,7 +2366,10 @@ class Segment2D(LinearEntity2D, Segment):
     Point2D(5/2, 2)
 
     """
-    def __new__(cls, p1, p2, **kwargs):
+
```
Comment (wsdookadr) [inline sympy/geometry/line.py:2369]: [The failing test](https://travis-ci.org/github/sympy/sympy/jobs/745076406#L325) is saying this training whitespace should be deleted.

**Review by wsdookadr — CHANGES_REQUESTED** [review]

**Comment by smichr** [comment]
-1 
I don't think this is a good idea and gave reasons on the issue page.

**Comment by smichr** [comment]
closing due to inactivity

**Pull request closed without merging**